client/.storybook/config.js

```diff
@@ -0,0 +1,9 @@
+import { addDecorator } from '@storybook/react';
+import themeDecorator from './themeDecorator';
+import { addParameters } from '@storybook/react';
```

Comment:
```suggestion
import { addDecorator, addParameters } from '@storybook/react';
import themeDecorator from './themeDecorator';
```
you can consolidate these

Reply:
Sure!